lib/Icon/IconAccessibilitySupport.js

```diff
@@ -0,0 +1,30 @@
+var
+	kind = require('enyo/kind');
+
+/**
+* @name IconAccessibilityMixin
+* @mixin
+*/
+module.exports = {
+
+	/**
+	* @private
+	*/
+	observers: [
+		{method: 'updateAccessibilityAttributes', path: ['icon']}
```

Comment:
You need to observe `small` rather than `icon` as the former drives the creation of the tapArea control